config.analysis

```diff
@@ -41,6 +48,9 @@ climo_yr1 = 6
 climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
+# start and end date for the analysis
+analysis_start_date = 0001-01-01_00:00:00
+analysis_end_date = 9999-01-01_00:00:00
```

Comment:
@xylar, I may be missing something here but what is the risk of opening too many files via xarray (which does lazy evaluation) and then segmenting in time following that step?   The reason I ask is because the above discussion may result in unnecessary code unless the loading step is demonstrated to be the bottleneck.  My understanding is that essentially xarray takes care of this heavy lifting behind the scenes.